doc/radosgw/notifications.rst

```diff
@@ -144,6 +148,20 @@ Request parameters:
   - "none": message is considered "delivered" if sent to broker
   - "broker": message is considered "delivered" if acked by broker (default)
 
+- AWS Endpoint
+
+ - push-endpoint: ``aws://sns.amazonaws.com`` for AWS SNS and ``aws://lambda.amazonaws.com`` for AWS lambda.
```

Comment:
it does not have to be these endpoints. don't we use the "schema" as an indication for the type of AWS endpoint.
e.g. `sns://<fqdn>[:port]` ?

Reply:
please use: `aws://<fqdn>[:port]`